test/fail_compilation/fail17722b.d

```diff
@@ -0,0 +1,13 @@
+/* TEST_OUTPUT:
```

Comment:
It's kept as two separate because the compiler doesn't issue an error for both when together in the same file.